### PR TITLE
convert Congrats and Certificate into functional components

### PR DESCRIPTION
**Pull request by davidsbailey**
this is a warm-up step for starting on [PLAT-1531].

## Testing story

* existing unit tests for each component
* verified that personalized certificates can be generated on http://localhost-studio.code.org:3000/congrats . this exercises the nameInputRef

[PLAT-1531]: https://codedotorg.atlassian.net/browse/PLAT-1531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/src/templates/Certificate.jsx [hunk 1]
```diff
@@ -23,152 +23,142 @@ const blankCertificates = {
...
+    certificateId,
+    randomDonorTwitter,
+    under13,
+    children
+  } = props;
 
-    const facebook = queryString.stringify({
-      u: certificateLink
-    });
+  const certificate = certificateId || 'blank';
```
Comment (megcrenshaw) [inline apps/src/templates/Certificate.jsx:61]: style nit: Could make `blank` the `defaultProp` for `certificateId` and avoid this, but it's old behavior

Comment (megcrenshaw) [inline apps/src/templates/Certificate.jsx:61]: style nit: Could make `blank` a `defaultProp` and avoid this, but I realize it's not what was there previously

**Review by megcrenshaw — APPROVED** [review]
Looks good! Glad the tests still work as-is

File: apps/src/templates/Certificate.jsx (hunk 1 above)
Comment (megcrenshaw) [reply apps/src/templates/Certificate.jsx:61]: (i.e. don't think it's worth a whole new build for it)

Comment (davidsbailey) [reply apps/src/templates/Certificate.jsx:61]: good call! I will include this in the next PR, where I am touching some of this code anyway.

**Review by tim-dot-org — APPROVED** [review]
👍

**Pull request merged**